framework/src/variables/MooseVariableFV.C

```diff
@@ -637,22 +591,7 @@ MooseVariableFV<OutputType>::getInternalFaceValue(const FaceInfo & fi,
 
   ADReal & value = *value_pointer;
 
-  if (_face_interp_method == Moose::FV::InterpMethod::VertexBased)
-  {
-    ADReal numerator = 0, denominator = 0;
-
-    for (const Node * const vertex : fi.vertices())
-    {
-      auto distance = (*vertex - fi.faceCentroid()).norm();
-
-      numerator += getVertexValue(*vertex) / distance;
-      denominator += 1. / distance;
-    }
-
-    value = numerator / denominator;
-  }
-  else
-    value = Moose::FV::linearInterpolation(*this, Moose::FV::makeCDFace(fi, correct_skewness));
+  value = Moose::FV::linearInterpolation(*this, Moose::FV::makeCDFace(fi, correct_skewness));
 
   return value;
```

Comment:
Might as well tighten up this white space (e.g. probably just delete the separating new-lines)

Reply:
Addressed!